dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/TeradataUtils.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2022-2023 Google LLC
+ * Copyright 2013-2021 CompilerWorks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.edwmigration.dumper.application.dumper.connector.teradata;
+
+import com.google.common.base.Preconditions;
+import java.util.Optional;
+
+class TeradataUtils {
+
+  static String formatQuery(String query) {
+    return query.replaceAll("\\s+", " ").replaceAll("\\( ", "(").replaceAll(" \\)", ")").trim();
```

Comment:
This is, how the method works. I've added a javadoc. I cannot return early if the query contains a single quote character, because there are already queries that contain such a character that are formatted by this method, e.g.
```
...FROM DBC.DatabasesV WHERE DBKind='U' ORDER BY PermSpace DESC...
```
generated here: 
https://github.com/google/dwh-migration-tools/blob/94c1e4005527c2fe4682ee2b6343543cac447e2b/dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/TeradataMetadataConnector.java#L206